app/experimenter/experiments/tests/test_email/test_email_legacy.py

```diff
@@ -254,3 +258,18 @@ def test_send_experiment_pausing_email(self):
             ],
         )
         self.assertIn("May 6, 2019", sent_email.body)
+
+    def test_send_experiment_comment_email(self):
+        experiment = ExperimentFactory.create(name="experiment1")
```

Comment:
make sure to set type to pref flip in here otherwise this will intermittently fail